pkg/popeye.go

```diff
@@ -134,16 +134,16 @@ func (p *Popeye) scannedGVRs(rev *client.Revision) internal.GVRs {
 		internal.RoGVR:  "rbac.authorization.k8s.io/v1/roles",
 		internal.RobGVR: "rbac.authorization.k8s.io/v1/rolebindings",
 		internal.IngGVR: "networking.k8s.io/v1/ingresses",
-		internal.PspGVR: "policy/v1/podsecuritypolicies",
+		internal.PspGVR: "policy/v1beta1/podsecuritypolicies",
 		internal.PdbGVR: "policy/v1/poddisruptionbudgets",
 		internal.HpaGVR: "autoscaling/v2/horizontalpodautoscalers",
 	}
 
 	if rev.Minor < 18 {
 		mm[internal.IngGVR] = "networking.k8s.io/v1beta1/ingresses"
 	}
-	if rev.Minor <= 21 {
-		mm[internal.PspGVR] = "policy/v1beta1/podsecuritypolicies"
+	if rev.Minor >= 25 {
+		delete(mm, internal.PspGVR)
```

Comment:
@derailed check here, do you think this approach for disabling the sanitizer is appropriate?

Reply:
@gargrag Thank you for the update Gaston! I think this is a dup of #239. It would be better to reverse the logic here and check the server version vs having to delete after the fact imho.